docs/src/gui/qtvcp-vismach.adoc

```diff
@@ -298,51 +346,54 @@ For a _parallel machine_ there may be several "base" parts.
 
+[source,python]
 ----
 model = Collection([base, saddle, head, carousel])
 ----
 
 == Other functions
 
-* `part = Color([colorspec], [part])` +
-  Sets the display color of the part. +
-  Note that unlike the other functions the part definition comes second
+*`part = Color([_colorspec_], [_part_])`*::
+  Sets the _display color of the part_. +
+  Note that unlike the other functions, the part definition comes second
   in this case. +
-  The colorspec consists of the three RGB values and an opacity. +
+  `_colorspec_`;; Three RGB values and opacity. +
   For example [1,0,0,0.5] for a 50% opacity red.
 
-* `myhud = Hud() +
-  myhud.show("Mill_XYZ")` +
-  Creates a heads-up display in the Vismach GUI to display such items as
-  axis positions, titles, or messages.
+*`myhud = Hud()`*::
+*`myhud.show("_Mill_XYZ_")`*::
+  Creates a _heads-up display_ in the Vismach GUI to display items such
+  as axis positions, titles, or messages.
 
-* `part = Capture()` +
+*`part = Capture()`*::
   This sets the current position in the model.
 
-* `main(model, tooltip, work, size=10, hud=myhud, rotation_vectors=None,
-  lat=0, lon=0)` +
+*`main(model, tooltip, work, size=10, hud=myhud, rotation_vectors=None, lat=0, lon=0)`*::
   This is the command that makes it all happen, creates the display etc.
-  if invoked directly from python3. +
-  Usually this file is imported by QtVCP and the window() object is
-  instantiated and embedded into another screen. +
-** _model_ should be a collection that contains all the machine parts. +
-** _tooltip_ and _work_ need to be created by Capture() to visualize
-   their motion in the backplot. +
-   See `mill_xyz.py` for an example of how to connect the tool tip to a
-   tool and the tool to the model.
-** Either _rotation_vectors_ or _latitude/longitude_ can be used to set
-   the original viewpoint and it is advisable to do as the default
-   initial viewpoint is rather unhelpfully from immediately overhead.
-** _size_ sets the extent of the volume visualized in the initial view.
-   hud refers to a head-up display of axis positions.
-
-== Basic structure of a QT Vismach script.
+  if invoked directly from Python. +
+  Usually this file is imported by QtVCP and the `window()` object is
+  instantiated and embedded into another screen.
+  `_model_`;; Should be a collection that contains all the machine parts.
+  `_tooltip_` and `_work_`;; Need to be created by `Capture()` to
+    visualize their motion in the backplot. +
+    See `mill_xyz.py` for an example of how to connect the tool tip to a
+    tool and the tool to the model.
+  `_size_`;; Sets the extent of the volume visualized in the initial
+    view. +
+    hud refers to a head-up display of axis positions.
+  `_rotation_vectors_` or `_lat, lon_`;; Can be used to set
+    the original viewpoint. +
+    It is advisable to do as the default initial viewpoint is rather
+    unhelpfully from immediately overhead.
+
+== Basic structure of a QtVismach script
```

Comment:
```suggestion
== Basic Structure of a QtVismach script
```

Reply:
Titles capitalization is really something we have to decide !

With my afternoon readings, I believe we should follow simple "Sentence" capitalization and only raise first character.

Reply:
If *S*tructures then also *S*cript.

We can fix this any time, but yes, we need to decide about it. Would leave that decision to the English natives, though.

Reply:
I'll follow https://docs.gitlab.com/ee/development/documentation/styleguide/#markdown-rule-md044proper-names-capitalization and https://docs.gitlab.com/ee/development/documentation/styleguide/#capitalization as they looks perfectly good to me.